class_project/ta.class_project_gen/git_docker_setup_assignment.md

```diff
@@ -0,0 +1 @@
+## Assignment Link - https://docs.google.com/document/d/18uWO5cCq68mhr5q_Cb2HSl28a8cKi24I-vguGo7R82M/edit?usp=sharing
```

Comment:
1) Is this needed in this PR?
2) Also it should go in the official gdrive dir so it can be reused or it should be a md. We need to reuse stuff for the next classes